test/specs/homePage.js

```diff
@@ -0,0 +1,15 @@
+describe('crossfadeYT Music Video', function(){
```

Comment:
Better `describe` please. What is the purpose of this test file, you should ask yourself?
Something like `Home Page Tests`